lib/ansible/modules/storage/netapp/na_ontap_volume.py

```diff
@@ -114,6 +114,19 @@
     - Allows a storage efficiency policy to be set on volume creation.
     version_added: '2.7'
 
+  unix_permissions:
+    description:
+    - Unix permission bits in octal or symbolic format.
+    - For example, 0 is equivalent to ------------, 777 is equivalent to ---rwxrwxrwx,both formats are accepted.
```

Comment:
Let me talk with the rest of the group and get back to you (on if we will support just Octal).

The reason we added support for both Octal (0755) and Symbolic (-rwxr-xr-x), is that the underlying command that user have to use today supports both. And we want to make the transition from the command to ansible to be as easy as possible. https://docs.netapp.com/ontap-9/index.jsp?topic=%2Fcom.netapp.doc.dot-cm-cmpr-920%2Fvolume__modify.html

Reply:
People who use the ONTAP API or console command today can use both Octal and Symbolic forms, we want to make sure that they can still do that in the ansible module. So we want to be able to support both so that they don't have to convert everything.

Reply:
understandable @carchi8py ping @gundalow. what are your thoughts. since its being provided both ways on ontap side, I am of the belief we ship.

Reply:
@carchi8py Thanks for the info. As this is something that ONTAP API users are used to it makes sense to support this in the module, thanks for the background.